boards/shields/mikroe_ble_tiny_click/mikroe_ble_tiny_click.overlay

```diff
@@ -15,5 +15,11 @@
 
 	bt_hci_uart: bt_hci_uart {
 		compatible = "zephyr,bt-hci-uart";
+		status = "okay";
+
+		da1453x {
+			compatible = "renesas,bt-hci-da1453x";
+			reset-gpios = <&mikrobus_header 1 GPIO_ACTIVE_HIGH>;
+		};
```

Comment:
Aren't you missing `status = "okay";` here? How exactly are you testing this?

Reply:
But isn't a node okay by default unless explicitly disabled? 🤔
https://docs.zephyrproject.org/latest/build/dts/intro-syntax-structure.html#id8

Reply:
> But isn't a node okay by default unless explicitly disabled? 🤔 https://docs.zephyrproject.org/latest/build/dts/intro-syntax-structure.html#id8

You're right. I got confused by the fact that the parent node to this one does have an explicit "okay".

Reply:
Agreed that it would be nice to be consistent as it has one wonder why this one node is seemingly different and it's confusing indeed

Reply:
Agree with you guys, updated as suggested.